react/components/EXPERIMENTAL_Table/Sections/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { forwardRef, Ref, PropsWithChildren } from 'react'
+import classNames from 'classnames'
+
+import EmptyState from '../../EmptyState/index.js'
+import { ORDER_CLASSNAMES } from '../constants'
+import { TABLE_HEADER_HEIGHT } from '../hooks/useTableMeasures'
+import Loading from './Loading'
+import {
+  E2ETestable,
+  ComposableWithRef,
+  HasMotion,
+  NativeTable,
+} from '../types'
+import { useMeasuresContext } from '../context/measures'
+import { useTestingContext } from '../context/testing'
+import { useLoadingContext } from '../context/loading'
+import Tbody, { ComposableTbody } from './Tbody'
+import Thead, { ComposableThead } from './Thead'
+
+type Props = PropsWithChildren<E2ETestable & HasMotion & NativeTable>
+
+function Sections(
+  { children, className, motion }: Props,
+  ref: Ref<HTMLTableElement>
```

Comment:
```suggestion
  ref: Ref<HTMLTableElement>,
```
Eslint didn't throw an error about this? :thinking: